### ISSUE TITLE
Web suggest links not accessible, with messy numbers

### ISSUE THREAD

**Issue by duzaichuan** [mnewt/counsel-web#3]
![counsel-web](https://user-images.githubusercontent.com/32022901/121316184-a39a8b80-c93b-11eb-88b8-2f7a86401664.png)

Manjaro, emacs 27.2. Any suggestions? Thanks.

**Comment by mnewt** [comment]
I've seen this before. DuckDuckGo tacks that `&rut=` business onto URLs when it doesn't like the User Agent (presumably as some sort of bot security measure). In the past, setting a User Agent of "Emacs" was sufficient to get rid of it. Now it seems it doesn't like that anymore.

I just tried to make this work with the User Agent that `url` sets, and it didn't work any better. For me, that is:

    User-Agent: URL/Emacs Emacs/28.0.50 (OpenStep; x86_64-apple-darwin20.4.0)

I tried a few random User Agent strings and some of them worked. But many that I expected to work didn't. So I don't know what their problem is. I'm starting to think that just living with that suffix and stripping it off is the better approach.

Edited to add: I'll push an update that strips that bit out.
